docs/reference/module-api/module-overview.md

```diff
@@ -287,6 +288,64 @@ modules.export = {
 };
 ```
 
+### `batchOperations`
+
+Piece types can use batch operations in the `batchOperations` cascade object property, which are actions editors can take on many selected pieces at once. Apostrophe has archive and restore (from the archive) batch operations by default, for example. New batch operations are added to a series of buttons in the piece type manager modal.
+
+#### `add`
+
+The `add` property is an object containing batch operation configurations. Each batch operation configurations should include the following properties:
+
+| Property | Description |
+| ------- | ------- |
+| `label` | A text label used for the batch operation button's readable label. |
+| `route` | A relative route path with leading slash where the manager modal should send the selected document IDs for processing. The route should be defined in [`apiRoutes`](#apiroutes-self). |
```

Comment:
After reading this I don't know what this route should look like. `/api/v1/something`? `/archive` but it gets appended to the `action` of the module?

For consistency with elsewhere in A2 and A3, if it starts with `/` it should be treated as the full `path` component of the URL, bypassing `action`. If it doesn't, then it should get appended to `action`, with a `/` between. One of many examples:

```
`${apos.i18n.action}/locale`
```

I don't know if this is a documentation bug or an implementation issue the documentation is faithfully describing.

Reply:
It's an  implementation issue the documentation is faithfully describing.